### PR TITLE
Improving console api

### PR DESCRIPTION
**Pull request by jacob-statsig**

### CODE REVIEW

**Comment by jacob-statsig** [comment]
Todo, 
- fixing a few issues with rapidoc
- updating some of the openApi documentation
- styling

**Comment by jasonwzm** [comment]
🚀 Deployed on https://62a10b537dbdfc006449283a--cozy-fox-0defec.netlify.app

File: docs/console-api/single-page.mdx [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+title: Single Page
+sidebar_label: Single Page
```
Comment (tore-statsig) [inline docs/console-api/single-page.mdx:3]: I'd call this: All Endpoints

File: docusaurus.config.js [hunk 2]
```diff
@@ -1,126 +1,137 @@
...
         },
         {
-          title: 'More',
+          title: "More",
           items: [
             {
-              label: 'Statsig Home',
-              href: 'https://statsig.com',
+              label: "Statsig Home",
+              href: "https://statsig.com",
```
Comment (tore-statsig) [inline docusaurus.config.js:42]: oh no, you have a different prettier configured

**Review by tore-statsig — APPROVED** [review]

File: docs/console-api/single-page.mdx (hunk 1 above)
Comment (jacob-statsig) [reply docs/console-api/single-page.mdx]: [f087697](https://github.com/statsig-io/docs/pull/294/commits/f08769708521e7857b3d8f90ea0b93eebac95c9d)

**Comment by tore-statsig** [comment]
1. Remove the "API Server" selector - there is only one choice (and probalby should only be one choice) so better to just hide it.

2. use https://statsigapi.net/console/v1 instead of api.statsig.com

3. Remove the Expand all / collapse all

4. I applied a key, but all the snippets are not updated - they still show  'STATSIG-API-KEY: console-xxxxXXXXxxxxXXXXxxxx' 

5. The "Try" button is small and hard to find - I thought the example data was for my project, but they I realized I had to click "try".  I'd make that a much bigger button, and left align it instead of right align it

6. Schema for "message" says "message: {recursive: message.yaml}"

7. Why is the url at the top "https://www.statsig.com/slack"?  I wouldnt have a slack url

8. Im not a fan of the highlight when you hover over the header, but thats not a big deal

9. Schema is a bit wrong for get all - for dynamic configs, it says data: {recursive: dynamic-config.json}

**Pull request merged**